assignment1.js

```diff
@@ -0,0 +1,80 @@
+/*Palindrome Permutation: Given a string, write a function to check if it is a permutation of a palin-
+drome. A palindrome is a word or phrase that is the same forwards and backwards. A permutation
+is a rearrangement of letters. The palindrome does not need to be limited to just dictionary words.
+EXAMPLE
+Input: T a c t Coa
+Output: T r u e ( p e r m u t a t i o n s : " t a c o c a t " , " a t c o e t a " , e t c . )*/
+
+function isPalindrome(query) {
```

Comment:
Technically, you're checking for permutations of palindromes, not just palindromes.
```suggestion
function isPalindromePermutation(query) {
```